frontend/src/auth/actions.ts

```diff
@@ -70,24 +70,28 @@ export const performLogin = (
       }
     })
     .catch((e: any) => {
-      e.json().then((feedback: any) => {
-        // If validation fails, return error to form through state
-        let error = feedback.detail;
-        // Don't provide individual feedback on each field for login
-        // If feedback.detail isn't set, then it means we got an
-        // response like this:
-        // {
-        //    username: ["This field is required.]
-        // }
-        // So manually set feedback error message.
-        if (!feedback.detail) {
-          error = 'Both the username and password fields are required.';
-        }
-        dispatch({
-          type: Types.LOGIN_FAILURE,
-          error
+      try {
+        e.json().then((feedback: any) => {
```

Comment:
@giovannicimolin, as the code below is treating `feedback` as an object, should we still use `any`?

Reply:
@lgp171188 If I were to set `feedback` to a type I'd need to create an interface to describe the possible fields inside the response feedback.
We know that the return of `.json()` is a `object`, but if I put that in the prop definition I get type errors because the members of this object are not defined.
I could define a generic object like this:
```
interface GenericJsonObject {
    [key: string]: any;
}
```
But I think it's overly complex for just catching and parsing the response here.